looker_deployer/commands/deploy_content.py

```diff
@@ -303,6 +359,8 @@ def main(args):
         args.target_base = 'Shared'
 
     sdk = get_client(args.ini, args.env)
+    global enabled_alerts
+    enabled_alerts = sdk.search_alerts(disabled="false")
```

Comment:
I believe you need `sdk.search_alerts(disabled=False, all_owners=True)` to ensure all alerts for all users are pulled in (same with schedules)